src/components/CreateTask.tsx

```diff
@@ -1,6 +1,10 @@
 import React, { useState } from 'react'
 
-export default function CreateTask({ addTask }) {
+type Props = {
+  addTask: Function
```

Comment:
From the eslint/recommended settings:
> Don't use `Function` as a type. The `Function` type accepts any function-like value.
> It provides no type safety when calling the function, which can be a common source of bugs.
> It also accepts things like class declarations, which will throw at runtime as they will not be called with `new`.
>
> If you are expecting the function to accept certain arguments, you should explicitly define the function shape.

Here's how to be more explicit with the shape of the function:
```suggestion
  addTask: (title: string) => void;
```